docs/docs/how-tos/upgrade-kubernetes-version.md

```diff
@@ -0,0 +1,155 @@
+:::
+
+:::warning
+Downgrading to a lower version of Kubernetes is dangerous and we strongly advise against it!
+:::
+
+<Tabs>
+  
+<TabItem label="GCP" value="gcp" default="true">
+
+Google Kubernetes Engine (GKE) cut their own platform specific version of Kubernetes that usually look something like: `1.26.7-gke.500`; this corresponds to a Kubernetes version of `1.26.17`.
+
+You can list the supported GKE Kubernetes versions by running the following `gcloud` command:
+
+```bash
+gcloud container get-server-config --region <region>
+```
+
+We recommend selecting a `validVersion` from the `STABLE` channel:
+
+```bash
+channels:
+- channel: RAPID
+  defaultVersion: 1.27.4-gke.900
+  validVersions:
+  - 1.28.1-gke.201
+  ...
+- channel: REGULAR
+  defaultVersion: 1.27.3-gke.100
+  validVersions:
+  - 1.27.4-gke.900
+  - 1.27.3-gke.1700
+  ...
+- channel: STABLE
+  defaultVersion: 1.27.3-gke.100
+  validVersions:
+  - 1.27.4-gke.900
+  - 1.27.3-gke.100
+  - 1.26.7-gke.500
+  ...
+  - 1.24.15-gke.1700
+  - 1.24.14-gke.2700
+```
+
+
+To upgrade your GKE cluster, update the `google_cloud_platform.kubernetes_version` field in your `nebari-config.yaml` to match one of these GKE Kubernetes versions. Then run `nebari deploy` to apply these changes. This deployment process might take as long as 30 minutes.
+
+:::info
+You will get a validation error if you try to select a Kubernetes version that is unsupported by GKE or a version higher than [`HIGHEST_SUPPORTED_K8S_VERSION`](https://github.com/nebari-dev/nebari/blob/91792952b67074b5c15c3b4009bde5926ca4ec6b/src/_nebari/constants.py#L11).
+:::
+
+Then repeat the above process one minor version at a time. You will get a similar error otherwise:
+
+```bash
+[terraform]: │ Error: googleapi: Error 400: Master cannot be upgraded to "1.26.7-gke.500": cannot upgrade the master more than a minor version at a time.
+```
+
+For more information about GKE upgrades, please refer to the [GKE documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/upgrading-a-cluster).
+
+</TabItem>
+
+
+<TabItem label="AWS" value="aws">
+
+The AWS Elastic Kubernetes Service (EKS) only requires that you supply the major and minor version of Kubernetes that you want. To specify Kubernetes version `1.26.7`, simply updated the `amazon_web_services.kubernetes_version` to `1.26`. Then run `nebari deploy` to apply these changes; you will get a validation error if you try to select a Kubernetes version that is unsupported by GKE or a version higher than `HIGHEST_SUPPORTED_K8S_VERSION`. This deployment process might take as long as 30 minutes.
+
+:::info
+You will get a validation error if you try to select a Kubernetes version that is unsupported by GKE or a version higher than [`HIGHEST_SUPPORTED_K8S_VERSION`](https://github.com/nebari-dev/nebari/blob/91792952b67074b5c15c3b4009bde5926ca4ec6b/src/_nebari/constants.py#L11).
+:::
+    
```

Comment:
Blurb for node pools:

In AWS, upgrading EKS will upgrade the control plane components 

In the AWS console, navigate to EKS and click on the name of your Kubernetes cluster (format will be {{project-name}}-{{namespace}}.  In the 'Compute' tab, scroll down to "Node Groups".  Any node groups which are behind will have an "Update Now" button by the "AMI release version" column values.  Click "Update Now" for each.  Each update may take 15 or more minutes depending on how many workloads need to be migrated, but they can be run simultaneously."

Reply:
Thanks @kenafoster :)